### ISSUE TITLE
eldoc incomplete

### ISSUE THREAD

**Issue by r2evans** [emacs-ess/ESS#806]
Related to #804. This time, eldoc just shows the function name and a colon, as in:

```
data.frame:
```

when I have `data.frame(` on an otherwise not-busy R prompt. (`(inferior-ess-available-p)` is `t`)

**Comment by vspinu** [comment]
I don't think this is reproducible. Can you check with the latest ESS?

**Comment by r2evans** [comment]
As I'm trying to upgrade, something else may be going on ... I'm at 2 minutes with `compiling c:/Users/r2/.emacs.d/elpa/ess-20190106.2002/ess-r-mode.el... (ess-r-end-of-function)` with a completely unresponsive emacs. I'll let it sit another few, but seems like something else might be interfering. (Only this package is being upgraded now.)

**Comment by r2evans** [comment]
Okay, I paused, then I got fed up and interrupted it with `C-g`. This prevent emacs from cleaning up, therefore leaving both versions of `ess-2018*` and `ess-2019*` installed, explaining my second paragraph in my first [eldoc question comment](https://github.com/emacs-ess/ESS/issues/804#issuecomment-451668065).

I was able to successfully run `C-u 0 M-x byte-recompile-directory` with no errors, and now the original problem goes away. I did not recompile last time, so it's likely something was in a bad state.

Ok, this problem solved, now I have another (not ESS). https://xkcd.com/1171/

Thanks for the help!
